### ISSUE TITLE
Use YAML instead of env. vars. for HW configuration.

### ISSUE THREAD

**Issue by arnoldandreasson** [cloudedbats/cloudedbats_wurb_2020#37]
Future development will need more configurable parts. YAML is better suited for that and integrates well with Python.
If the default YAML-file is stored as a Git controlled file and the detector check for a YAML-file in /home/pi/wurb_settings before the default location then there will be no problems with the git pull command.

**Comment by slowglow** [comment]
I strongly agree with the need to move the configuration settings from environment variables to something else.  I will certainly use them in the future.

Not wishing to start a flame war here, because as am not knowlegeable enough and have no preference  about either of them, but have you considered TOML vs YAML? I am only asking, because I recently tried to use `poetry` for python project management and it uses TOML file, which was advertised as much better than 'requrements.txt` and anything else before :-).

Putting the visual difference aside, what would be the top three technical arguments to choose one over the other (YAML vs. TOML)? 

I know that sometimes these discussions can get bogged down in a typical example of bike-shedding and I accept that developers personal experience and preference is valid reason to select one over the other.  Still, it could be useful to have a quick evidence-driven comparison.
